### DISCUSSION TITLE
Strict

### DISCUSSION THREAD

**Discussion by robert-abram** [spatie/laravel-data#93] [Q&A]
Is it possible to have #[Strict] Data object like we have in laravel dto ??

```php
use Spatie\DataTransferObject\Attributes\Strict;
#[Strict]
class MyData extends Data{
}
```
This not working

**Comment by rubenvanassche** [comment]
Hi @abrampl,

At the moment we do not have such a feature. But I would be open for a well tested PR that adds this functionality!
